### PR TITLE
feat(uploads): add resin tracking for uploads manager

### PR DESCRIPTION
**Pull request by rosepierce**
Adding resin tracking for retrying failed uploads and expanding/collapsing the uploads manager.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @rosepierce has signed the CLA. Thanks for the pull request!

File: src/elements/content-uploader/OverallUploadsProgressBar.js [hunk 1]
```diff
@@ -73,6 +75,10 @@ const OverallUploadsProgressBar = ({ percent, view, onClick, onKeyDown, isDraggi
     return (
         <div
             className="bcu-overall-progress-bar"
+            data-resin-target={classNames({
```
Comment (jstoffan) [inline src/elements/content-uploader/OverallUploadsProgressBar.js]: Should we use `classNames` for non-CSS purposes? I had to look at this a few times to realize it wasn't setting actual class names.

File: src/elements/content-uploader/__tests__/__snapshots__/ItemAction-test.js.snap [hunk 2]
```diff
@@ -18,6 +18,7 @@ exports[`elements/content-uploader/ItemAction should render correctly with STATU
     theme="default"
   >
     <PlainButton
+      data-resin-target=""
```
Comment (jstoffan) [inline src/elements/content-uploader/__tests__/__snapshots__/ItemAction-test.js.snap]: It seems like the attribute shouldn't be applied if the value is empty.

File: src/elements/content-uploader/ItemAction.js [hunk 3]
```diff
@@ -52,7 +54,7 @@ const ItemAction = ({ status, onClick, intl, isFolder = false }: Props) => {
     return (
         <div className="bcu-item-action">
             <Tooltip position="top-left" text={tooltip}>
-                <PlainButton onClick={onClick} type="button">
+                <PlainButton data-resin-target={target} onClick={onClick} type="button">
```
Comment (mickr) [inline src/elements/content-uploader/ItemAction.js]: Should this be used to track other states as well? If it isn't retrying should that data attribute be on the button?

**Pull request merged**